docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks.md

```diff
@@ -9,24 +9,74 @@ modeled in [Decision Model and Notation](https://www.omg.org/dmn/) (DMN).
 
 ![task](assets/business-rule-task.png)
 
-Business rule tasks behave exactly like [service tasks](/components/modeler/bpmn/service-tasks/service-tasks.md). Both
-task types are based on jobs and [job workers](/components/concepts/job-workers.md).
+:::note
+Camunda Cloud supports alternative task implementations for the business rule task. If you want to
```

Comment:
:x: With the release `1.4.0`, we should use the term `Camunda Platform` instead :wink: 

```suggestion
Camunda Platform supports alternative task implementations for the business rule task. If you want to
```